wordpress-mobile/WordPress-iOS
# Change WordPress / Jetpack launch screen

Closes #19325

### Description
This PR updates WordPress and Jetpack launch screens to match the design in Figma qk5s4q4mZsXknwhMWo2vIY-fi-3098%3A11890

### Test Instructions:
Do the following for both WordPress and Jetpack apps:
1. Force close the app if it's already running.
2. Run the app. You should see the launch screen briefly.

| Target | iPhone | iPad |
| --- | --- | --- |
| **WordPress** | <img src="https://user-images.githubusercontent.com/9609223/191992021-20f3bde1-14f0-4c5d-8c02-f7d481438638.png" width="300px"/> | <img src="https://user-images.githubusercontent.com/9609223/191992719-ce0c5b72-9813-4a63-bddb-3c9c7864f4e2.png" width="300px"/> |
| **Jetpack** | <img src="https://user-images.githubusercontent.com/9609223/191993028-6d1af227-f1ea-44d2-ab13-04ebe84ac2c9.png" width="300px" /> | <img src="https://user-images.githubusercontent.com/9609223/191993313-bbb3ed3c-4cd4-45d3-b94d-79c1418f0e3f.png" width="300px"/> |

### Regression Notes
1. Potential unintended areas of impact
None.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
None.

4. What automated tests I added (or what prevented me from doing so)
None.

### PR submission checklist:
